Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:latest
+
+# Install Python and necessary dependencies
+RUN apt-get update && \
+    apt-get install -y python3 python3-pip git && \
+    rm -rf /var/lib/apt/lists/* && \
+    ln -s /usr/bin/python3 /usr/bin/python && \
+    pip3 install setuptools
```

Comment:
The teams that I work with take this approach all the time. Do I know whether or not it is needed?  I do not know, but it is such a light command that I never worried about it. It can be removed.